src/components/Navbar.jsx

```diff
@@ -1,45 +1,51 @@
 import { useState } from "react";
-import { close, logo, menu } from "../assets";
+import { close, menu } from "../assets";
+import incitext from "../assets/incident-text.png";
 import { navLinks } from "../constants";
+import unstop from "../assets/unstop-logo.png";
 
 const Navbar = () => {
   const [toggle, setToggle] = useState(false);
 
   return (
-    <nav className="w-full flex justify-between items-center navbar">
+    <nav className="w-full z-50 md:p-3 py-4 flex justify-between items-center backdrop-blur-md fixed top-0">
       {/* Logo */}
       <a href="#home">
         <img
-          src={logo}
+          src={incitext}
           alt="Incident 2023"
-          className="w-[80px] h-[80px] mt-2"
+          className="xl:h-[43px] md:h-[33px] sm:h-[28px] h-[22px] justify-end md:mx-20 mx-10"
```

Comment:
Can reduce the default to `mx-4` (the margin of the logo from the left and closed menu from the right isn't the same) and/or make it `justify-start`

Reply:
Fixed it